tests/components/dropdown.spec.js

```diff
@@ -28,7 +28,7 @@ describe("dropdown", async () => {
         // Without async iterators, just use a for loop.
         for (let i = 0; i < dds.length; i++) {
             Array.from(dds[i].children)
-                .find(node => node.tagName === "BUTTON" && node.classList.contains("dropdown-toggle"))
+                .find(node => node.tagName === "BUTTON" && node.id.includes("_BV_toggle_"))
```

Comment:
I had to change this to look for the id instead of the class name due to the new `hide-toggle` prop. i'm not familiar with how bootstrap vue generates the id so hopefully this is suitable?

Reply:
Here's how it is done:

If an ID is passed to the `id` prop of the `<b-dropdown>`, then that ID is used as the base and `__BV_toggle_` is appended.  If no ID is provided, then Bootstrap-Vue (on client only), we use the Vue unique UID (each vue VM is assigned a unique instance I on the client) to generate an ID string and `__BV_toggle_`is ppended to that.  The ID generation is handled by the id mixin's `safeId()` method.